satellite/satellitedb/dbx/gen.go

```diff
@@ -13,17 +13,9 @@ import (
 //go:generate dbx.v1 golang -d postgres -d sqlite3 satellitedb.dbx .
 
 func init() {
-	// catch dbx errors
-	c := errs.Class("satellitedb")
+	// do not hide the actual error, necessary for other logic to work
 	WrapErr = func(e *Error) error {
-		switch e.Code {
-		case ErrorCode_NoRows:
-			return e.Err
-		case ErrorCode_ConstraintViolation:
```

Comment:
if I recall correctly, this was added because when there is a constraint violation, no extra information is printed to let us know details about it. Which makes those types of errors more difficult to debug if we remove this case

Reply:
I agree, but currently this is causing bugs in rest of the system. I'm fine with adding this back later, but at the moment I'm unsure where all the bugs are.

Reply:
any chance we can preserve the stack trace though? no stack trace is going to make database errors way harder to debug

Reply:
This means going through all the error handling of our database code. I would rather have the safer behavior be the default for now.

Reply:
fwiw, the reason dbx has WrapErr is to support stack trace attaching via something like zeebo/errs. i'd rather have code that depends on err equality do something else instead of removing stack traces

Reply:
also, what safer behavior are you expecting? do you know we're doing err equality checking with postgres errors somewhere? isn't that easier to grep for than removing stack traces?

Reply:
This probably means having flaky tests for a few days.

Reply:
how come?

Reply:
discussed on a chat: decided we'll just fix the problems we know about and keep the error wrapping. stack traces are so useful.